docker/core/Dockerfile

```diff
@@ -1,29 +1,35 @@
+# FIXME use alpine
 FROM node:14
 
 RUN mkdir -p /usr/app/src \
 	&& mkdir -p /usr/app/media \
-	&& mkdir -p /usr/app/scripts
+	&& mkdir -p /usr/app/scripts \
+	&& apt-get update \
+  && apt-get install netcat -y
 
-RUN apt-get update
-RUN apt-get install netcat -y
+WORKDIR /usr/app/src
 
-COPY ./docker/core/bootstrap.sh /usr/app/scripts/bootstrap.sh
-COPY ./docker/core/wait.sh /usr/app/scripts/wait.sh
-COPY . /usr/app/src
-RUN mkdir /usr/app/src/state
+COPY package.json /usr/app/src
 
 RUN chown -R node:node /usr/app
 
-WORKDIR /usr/app/src
-
 USER node
 
 ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
 ENV PATH="/home/node/.npm-global/bin:${PATH}"
 
-RUN npm install -g --loglevel warn nodemon bunyan && npm cache clean --force
-RUN npm install --loglevel warn
+# FIXME no need for nodemon
```

Comment:
I would keep nodemon for dev, but we can remove it in prod I guess

Reply:
yes as mentioned above, there's the need for a discussion. Now I am trying to close as much as possible :D